.github/workflows/lint_python.yml

```diff
@@ -1,30 +0,0 @@
-name: lint Python
-on: [pull_request]
-jobs:
-  lint_python:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v3
-      - uses: pre-commit/action@v3.0.0
-      - run: pip install --upgrade pip wheel
-      - run: pip install -U bandit codespell flake8 isort pytest pyupgrade tox
-
-      - name: bandit
-        run: bandit -r . || true
-
-      - name: Run CodeSpell
-        run: codespell --ignore-words-list="brane,gool,ist,sherif,wil" --quiet-level=2 --skip="*.key" || true
-      - run: pip install -r requirements.txt || true
-
-      - name: Run tox
-        run: tox || true
-
-      - name: Run pytest
-        run: pytest . || true
-
-      - name: Test pytest with doctest
-        run: pytest --doctest-modules . || true
```

Comment:
Those three were not even working anyway, and I'm not sure why they are on the `linter` workflow...